src/main/java/org/radarcns/managementportal/Subject.java

```diff
@@ -0,0 +1,204 @@
+                   @JsonProperty("sources") List<Source> sources,
+                   @JsonProperty("email") String email,
+                   @JsonProperty("project") Project project) {
+        this.login = login;
+        this.externalId = externalId;
+        this.externalLink = externalLink;
+        this.attributes = attributes;
+        this.status = status;
+        this.sources = sources;
+        this.email = email;
+        this.project = project;
+    }
+
+    @JsonProperty("id")
+    public String getLogin() {
+        return login;
+    }
+
+    public Integer getExternalId() {
+        return externalId;
+    }
+
+    public URL getExternalLink() {
+        return externalLink;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public Map<String, String> getAttributes() {
+        return attributes;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public List<Source> getSources() {
+        return sources;
+    }
+
+    public Project getProject() {
+        return project;
+    }
+
+    /**
+     * Returns the Human Readable Identifier associated with this subject.
+     * @return {@link String} stating the Human Readable Identifier associated with this subject
+     */
+    @JsonIgnore
+    public String getHumanReadableIdentifier() {
+        return getAttribute(HUMAN_READABLE_IDENTIFIER_KEY);
+    }
+
+    /**
+     * Gets the project attribute (e.g. tag) associated with the given {@link String} key.
+     * @param key {@link String} tag key
+     * @return {@link String} value associated with the given key
+     */
+    @JsonIgnore
+    public String getAttribute(String key) {
+        return attributes.get(key);
+    }
+
+    /**
+     * Generates the {@link JsonNode} representation of the current instance.
+     * @return {@link JsonNode} serialising this object
+     * @throws IOException in case the serialisation cannot be complete
+     */
+    @JsonIgnore
+    public JsonNode getJson() throws IOException {
+        return new ObjectMapper().readTree(getJsonString());
+    }
+
+    /**
+     * Generates the JSON {@link String} representation of the current instance.
+     * @return {@link String} serialising this object
+     * @throws IOException in case the serialisation cannot be complete
+     */
+    @JsonIgnore
+    public String getJsonString() throws IOException {
+        return new ObjectMapper().writeValueAsString(this);
+    }
+
+    /**
+     * Converts the {@link String} to a {@link Subject} entity.
+     * @param response {@link String} that has to be converted
+     * @return {@link Subject} stored in the {@link String}
+     * @throws IOException in case the conversion cannot be computed
+     */
+    @JsonIgnore
+    public static Subject getObject(String response) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
You could use a single `static final ObjectMapper` and configure it in a `static` block, instead of recreating + rebuilding it in every call.